--- icu/source/tools/toolutil/pkg_genc.c	2015-03-27 21:10:56.000000000 +0000
+++ icu/source/tools/toolutil/pkg_genc.c.patched	2016-08-13 02:21:40.573442401 +0000
@@ -5,6 +5,10 @@
  */
 #include "unicode/utypes.h"
 
+#ifndef ELF64_ST_INFO
+#   define ELF64_ST_INFO(bind, type)(((bind) << 4) + ((type) & 0xf))
+#endif
+
 #if U_PLATFORM_HAS_WIN32_API
 #   define VC_EXTRALEAN
 #   define WIN32_LEAN_AND_MEAN
